Makefile

```diff
@@ -6,3 +6,9 @@ lint:
 
 test:
 	go test ./...
+
+# Sync the Github organization according to config in org/
+# Must be run by a Github admin (generally in CI)
+sync-org:
+	/app/prow/cmd/peribolos/app.binary --fix-org --fix-org-members --fix-teams --fix-team-members \
```

Comment:
is /app/prow/cmd part of the ci container? I don't recall seeing this - seems like a very odd path for ci..

Cheers
-streve

Reply:
We don't use build-tools for this. Having an obscure path is almost a feature, as it will prevent anyone from accidentally running this

Reply:
We use upstream k8s prow images